.rubocop_todo.yml

```diff
@@ -142,28 +137,34 @@ Layout/SpaceInsidePercentLiteralDelimiters:
 
 # Offense count: 2
 # Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle.
+# Configuration parameters: EnforcedStyle, EnforcedStyleForEmptyBrackets.
 # SupportedStyles: space, no_space
+# SupportedStylesForEmptyBrackets: space, no_space
 Layout/SpaceInsideReferenceBrackets:
   Enabled: false
 
 # Offense count: 55
 Lint/AmbiguousBlockAssociation:
   Enabled: false
 
-# Offense count: 113
+# Offense count: 121
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 
-# Offense count: 89
+# Offense count: 1
+# Configuration parameters: AllowSafeAssignment.
+Lint/AssignmentInCondition:
```

Comment:
@bastilian could you have a look there? It's app/services/report_scanner/puppet_report_scanner.rb:5

Reply:
Sorry for the delay. Here: https://github.com/theforeman/foreman/pull/5341